packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -360,14 +367,25 @@ class Scaffold extends StatefulWidget {
   final List<Widget> persistentFooterButtons;
 
   /// A panel displayed to the side of the [body], often hidden on mobile
-  /// devices.
-  ///
+  /// devices. Swipes in from either left-to-right (LTR languages) or
+  /// right-to-left (RTL languages)
```

Comment:
This is a tiny nit, but I'd replace "LTR languages" with "[TextDirection.ltr]" (and the same for "RTL languages") so that the dartdocs link back to the TextDirection docs, which explain more about what's going on here.

Reply:
Fixed